examples/visco_elasticity/spring_kelvin_model.py

```diff
@@ -0,0 +1,148 @@
+    time integration: backward Euler
+
+    '''
+    def __init__(self, parameters: dict[str, float], constraint: Constraint):
+        self._constraint = constraint
+        self.E0 = parameters["E0"] # elastic modulus
+        self.E1 = parameters["E1"] # visco modulus
+        self.tau = parameters["tau"] # relaxation time == eta/(2 mu1) for 1D case eta/E1
+        if Constraint.UNIAXIAL_STRESS:
+            self.nu = 0.0
+        else:
+            self.nu = parameters["nu"] # Poisson's ratio
+
+        # lame constants (need to be updated if time dependent material parameters are used)
+        self.mu0 = self.E0 / (2.0 * (1.0 + self.nu))
+        self.lam0 = self.E0 * self.nu / ((1.0 + self.nu) * (1.0 - 2.0 * self.nu))
+        self.mu1 = self.E1 / (2.0 * (1.0 + self.nu))
+
+        self.I2 = np.zeros(self.stress_strain_dim, dtype=np.float64)  # Identity of rank 2 tensor
+        self.compute_elasticity() # initialize elasticity tensor
+
+    def compute_elasticity(self):
+        match self._constraint:
+            case Constraint.FULL:
+                self.D_0 = np.array(
+                    [
+                        [2.0 * self.mu0 + self.lam0, self.lam0, self.lam0, 0.0, 0.0, 0.0],
+                        [self.lam0, 2.0 * self.mu0 + self.lam0, self.lam0, 0.0, 0.0, 0.0],
+                        [self.lam0, self.lam0, 2.0 * self.mu0 + self.lam0, 0.0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, 2.0 * self.mu0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, 0.0, 2.0 * self.mu0, 0.0],
+                        [0.0, 0.0, 0.0, 0.0, 0.0, 2.0 * self.mu0],
+                    ]
+                )
+                self.I2[0] = 1.0
+                self.I2[1] = 1.0
+                self.I2[2] = 1.0
+
+            case Constraint.PLANE_STRAIN:
+                self.D_0 = np.array(
+                    [
+                        [2.0 * self.mu0 + self.lam0, self.lam0, self.lam0, 0.0],
+                        [self.lam0, 2.0 * self.mu0 + self.lam0, self.lam0, 0.0],
+                        [self.lam0, self.lam0, 2.0 * self.mu0 + self.lam0, 0.0],
+                        [0.0, 0.0, 0.0, 2.0 * self.mu0],
+                    ]
+                )
+                self.I2[0] = 1.0
+                self.I2[1] = 1.0
+
+            case Constraint.PLANE_STRESS:
+                self.D_0 = (
+                        self.E0
+                        / (1 - self.nu ** 2.0)
+                        * np.array(
+                    [
+                        [1.0, self.nu, 0.0, 0.0],
+                        [self.nu, 1.0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, (1.0 - self.nu)],
+                    ]
+                )
+                )
+                self.I2[0] = 1.0
+                self.I2[1] = 1.0
+
+            case Constraint.UNIAXIAL_STRESS:
+                self.D_0 = np.array([[self.E0]])
+                self.I2[0] = 1.0
+            case _:
```

Comment:
because I don't think that we need that and I don't checked if that will influence the visco part. For now I would like to exclude it. So far its only included in the elastic model... 
I can also exclude the 2D cases if that would then look better ;). For AM we need the 3D case and the 1D is more for testing issues.

Reply:
Do what you fell is best, as long as the tests run